src/models/thelio-r3-n1/repairs.md

```diff
@@ -0,0 +1,569 @@
+
+![Front crossbar screws](./img/crossbar-screws-front.webp)
+
+4. Unscrew the two back screws holding the top crossbar onto the back of the chassis.
+
+![Back crossbar screws](./img/crossbar-screws-back.webp)
+
+5. Unscrew the two screws holding the CPU duct onto the top crossbar.
+
+![CPU duct screws (top)](./img/cpu-duct-screws-top.webp)
+
+6. Lift the top crossbar up and out of the chassis.
+7. Unscrew the four SATA backplane screws from the drive cage, then push the SATA backplane out through the top of the drive cage.
+
+![SATA backplane screws](./img/sata-backplane-screws.webp)
+
+8. Place the new SATA backplane into the drive cage and secure it with the four SATA backplane screws, then reinstall the drive cage (if necessary).
+9. Plug the 4-pin Berg power connector and SATA data cables into the SATA backplane.
+    - The order of the SATA data cables shouldn't matter as long as your operating system and software is configured to address disks by UUID (e.g. `/dev/disk/by-id/...`) instead of by letter (e.g. `/dev/sd_`).
+    - By default, the `DATA1` port connects to the innermost SATA connector on the motherboard, and the `DATA0` port connects to the outermost SATA connector on the motherboard.
+10. Reinstall the CPU shroud (if necessary), any 2.5" drives that were removed, the 2.5" drive cage cover, and the top case.
+
+## Replacing the top I/O:
+
+Thelio Spark B1-N2 includes a top I/O module providing audio and USB ports. If the top ports are damanged or become defective, they can be replaced using the steps below.
+
+**Part numbers:**
+
+- The top I/O module's part number is `MYS7523 Rev. 4.0`.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 30 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the top I/O:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), and [remove the GPU](#replacing-the-gpu).
+2. Unplug the `JAUD1` header at the bottom-left corner of the motherboard.
+
+![Top I/O audio header](./img/motherboard-headers-audio.webp)
+
+3. Unplug the `JUSB1` header (highlighted cyan below) and `JUSB2` header (highlighted green below) along the right edge of the motherboard.
+    - The `JUSB1` header is behind the CPU duct, but it can be unplugged without removing the duct. The duct has been removed for greater visibility in the below photo.
+
+![Top I/O USB headers](./img/motherboard-headers-usb.webp)
+
+4. Unscrew the two front screws holding the top I/O board onto the chassis.
+
+![Top I/O screws](./img/top-io-screws.webp)
+
+5. Remove the top I/O board from the system.
+6. Insert the new top I/O board into the system and screw it into place.
+7. Reconnect the audio and USB headers.
+8. Replace the side brace (if necessary), CPU shroud, and top case.
+
+## Troubleshooting the power button:
```

Comment:
I think it would be helpful to mention Io v2's built-in power button LED as a troubleshooting tool. It is just below the power button, covered by the green line around the button, in the picture below. Should be helpful for indicating whether a connection issue is between the main button receptacle and Io, or between Io and the motherboard.

Reply:
I can move the line and mention the LED, but how does that help troubleshoot any differently than just pushing the button does?

Reply:
I think there should be a mention of it, regardless of whether it's better or worse for troubleshooting.

Reply:
I've incorporated the LED into the section.

Reply:
I'm going to update the Spark section to include this, too.